content/docs/2.0/scalers/influxdb.md

```diff
@@ -0,0 +1,61 @@
++++
+title = "Influxdb"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on influxdb queries"
++++
+
+### Trigger Specification
+
+This specification describes the `influxdb` trigger that scales based on the results of a flux query.
+
+```yaml
+triggers:
+  - type: influxdb
+    metadata:
+      authToken: some-auth-token
+      organizationName: influx-org
```

Comment:
Does this scaler only support InfluxDB Cloud or not? Because then we should rename the scaler imo

Reply:
No it supports open source version as well

Reply:
https://docs.influxdata.com/influxdb/v2.0/organizations/

Reply:
Makes sense, thanks!